templates/pages/assets/cable.html.twig

```diff
@@ -132,7 +132,8 @@
                               'socketmodels_id': item.fields['socketmodels_id_endpoint_' ~ side],
                               'itemtype': item.fields['itemtype_endpoint_' ~ side],
                               'items_id': item.fields['items_id_endpoint_' ~ side]
-                           }
+                           },
+                           'used': call('Glpi\\Socket::getSocketAlreadyLinked', [item.fields['itemtype_endpoint_' ~ side], item.fields['items_id_endpoint_' ~ side]])
```

Comment:
Maybe it will fix the issue reported. To be tested.
```suggestion
                           'used': item.fields['items_id_endpoint_' ~ side] is not empty ?call('Glpi\\Socket::getSocketAlreadyLinked', [item.fields['itemtype_endpoint_' ~ side], item.fields['items_id_endpoint_' ~ side]] : [])
```